configure.ac

```diff
@@ -1054,8 +1054,8 @@
             [enable_af_packet="no"],
             [[#include <sys/socket.h>
               #include <linux/if_packet.h>]])
-        AC_CHECK_DECL([PACKET_FANOUT],
```

Comment:
Hi,
Yes, no functional change and for now no impact that can be foreseen.
Le 21 juil. 2015 11:38 PM, Victor Julien <notifications@github.com> a écrit :In configure.ac:
> @@ -1054,8 +1054,8 @@
>              [enable_af_packet="no"],
>              [[#include <sys/socket.h>
>                #include <linux/if_packet.h>]])
> -        AC_CHECK_DECL([PACKET_FANOUT],

I see. So really the whole test is sort of useless?

—Reply to this email directly or view it on GitHub.